indico/modules/events/tracks/operations.py

```diff
@@ -31,4 +32,21 @@ def create_track(event, data):
     logger.info('Track %r created by %s', track, session.user)
     event.log(EventLogRealm.management, EventLogKind.positive, 'Tracks',
               'Track "{}" has been created.'.format(track.title), session.user)
+    flash(_('Track "{}" has been created.').format(track.title), 'success')
     return track
+
+
+def update_track(track, data):
+    track.populate_from_dict(data)
+    db.session.flush()
+    logger.info('Track %r modified by %s', track, session.user)
+    track.event_new.log(EventLogRealm.management, EventLogKind.change, 'Tracks',
+                        'Track "{}" has been modified.'.format(track.title), session.user)
+    flash(_('Track "{}" has been modified.').format(track.title), 'success')
+    return track
+
+
+def delete_track(track):
+    db.session.delete(track)
+    logger.info('Track deleted by %s: %s', session.user, track)
```

Comment:
`%r` for both (makes no difference but it's cleaner since we want the repr anyway)